IdentityCore/tests/MSIDThrottlingServiceIntegrationTests.m

```diff
@@ -1015,570 +1015,554 @@ - (void)testMSIDThrottlingServiceIntegration_ThrottledNonSSOSilentRequestWithUIR
 
 - (void)testMSIDThrottlingServiceIntegration_SSOSilentRequestWith429MSIDError_ShouldBeThrottledSuccessfully_AndThenUnThrottledUponExpiration
 {
-   if (@available(macOS 10.15, *))
```

Comment:
This class appears to have a lot of changes because after the `if (@available(macOS 10.15, *))` checks have been removed the indentation has changed. In truth only the `if (@available(macOS 10.15, *))` have been removed